python/GangaLHCb/test/GPI/TestExternalGaudiExec.py

```diff
@@ -110,6 +110,10 @@ def testSubmitJobComplete(self):
         from Ganga.GPI import jobs
 
         j = jobs[-1]
+        j = j.copy()
+        j.unprepare()
+        j.prepare()
+        j.submit()
```

Comment:
This concerns me a bit as the persistency here is crucial to 90% of the tests I've written. Nothing should have changed in this regard. You're still accessing the job from the repo, is the next step of sleep until hanging? In which case this is likely not due to persistency problems.